src/funding/card/config.jsx

```diff
@@ -90,85 +82,94 @@ export function getCardConfig() : FundingSourceConfig {
-                }
+        logoColors: {
+            [ BUTTON_COLOR.WHITE  ]:  BUTTON_COLOR.BLACK,
+            [ DEFAULT ]:              BUTTON_COLOR.WHITE
+        },
 
-                const vendorConfig = vendors[name];
+        handleClick: true,
 
-                if (!vendorConfig) {
-                    return null;
-                }
+        Logo: ({ onClick, nonce, content, logoColor, locale = {} }) => {
+            const { lang } = locale;
+            const isRTLLanguage = lang === 'he' || lang === 'ar' || lang === 'fa';
 
-                const { Label } = vendorConfig;
-                
-                return (
-                    <div
-                        class={ `${ CLASS.CARD } ${ CLASS.CARD }-${ name }` }
-                        onClick={ event => onClick(event, { card: name }) }
-                        tabindex='0'
-                        role='button'
+            return (
+                <div>
+                    <style
+                        nonce={ nonce }
+                        innerHTML={ `
+                            .paypal-logo-color-black #black-credit-card-button {
+                                color: #000;
+                                blackground: #fff;
+                            }
+
+                            .paypal-logo-color-white #black-credit-card-button { 
+                                color: #fff;
+                            }
+
+                            #black-credit-card-button {
+                                background: transparent;
+                                font-size: 1.15em;
+                            }
+
+                            #black-credit-card-button img {
+                                display: inline-block;
+                                height: 1.3em;
+                                vertical-align: middle;
+                                padding: 0 9px;
+                            }
+
+                            .dom-ready #black-credit-card-button div {
+                                -webkit-animation: show-text 1.0s 0s forwards;
+                                -moz-animation:    show-text 1.0s 0s forwards;
+                                -o-animation:      show-text 1.0s 0s forwards;
+                                animation:         show-text 1.0s 0s forwards;
+                                vertical-align: middle;
+                                overflow: hidden;
+                                display: inline-block;
+                                max-width: 0%;
+                                opacity: 0;
+                                position: static;
+                                visibility: visible;
+                            }
+
+                            #black-credit-card-button div {
+                                position: absolute;
+                                visibility: hidden;
+                            }
+                        ` } />
+                    <button
```

Comment:
Can we avoid having an extra button element here? We already have a parent element with `role=button`